app/views/work_locks/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<turbo-frame id='workAdminSection'>
+  <%= form_with url: locks_path(params[:id]), method: :put, data: {turbo: false} do |form| %>
```

Comment:
Why are you disabling turbo here?

Reply:
After the item is locked/unlocked by the controller, there is a re-direct and a flash message is shown, so we break out of the turbo frame for that.  Same pattern as the similarly implemented "change owner" feature: 

https://github.com/sul-dlss/happy-heron/blob/main/app/controllers/work_owners_controller.rb#L12-L23
https://github.com/sul-dlss/happy-heron/blob/main/app/views/work_owners/edit.html.erb#L2

Reply:
Ah, okay, I must not have the latest code when I looked.